hera_cal/abscal_funcs.py

```diff
@@ -874,6 +876,10 @@ def UVData2AbsCalDict(datanames, pol_select=None, pop_autos=True, return_meta=Fa
     # turn into datacontainer
     data, flags = DataContainer(d), DataContainer(f)
 
+    # turn into weights
+    if return_wgts:
+       flags = DataContainer(odict(map(lambda k: (k, (~flags[k]).astype(np.float)), flags.keys())))
```

Comment:
Maybe it would be nice to override the __invert__ operator on the DataContainer object so it's easy to go between weights and flags? This can be done at some later time if it seems worthwhile to you and others.

Reply:
thats an interesting idea.. seems like there is some ambiguity there, because you might want to invert a flag DC to get the inverse flags, but still want them as booleans, whereas w/ weights you want to invert and change `dtype`...

Reply:
we could add a `convert_to_wgts` and a `convert_to_flgs` method, but I'm happy with this one-liner for now...

Reply:
That's a good point. Sounds good to me!